docs/source/convex-hull.rst

```diff
@@ -0,0 +1,46 @@
+Convex Hull
+===========
+
+Used to find the smallest convex polygon that envolves a set of points in a 2D plane.
+
+.. note::
+
+    Tested on: URI1982
+
+.. code-block:: cpp
+
+    
+	struct Point{
+	    int x, y;
+	    Point(){}
+	    Point(int _x, int _y){
+	        x = _x;
+	        y = _y;
+	    }
+	    bool operator <(const Point &p)const {
+	        return x < p.x || (x == p.x && y < p.y);
+	    }
+	};
+
+	ll cross(const Point &O, const Point &A, const Point &B){
+	    return (A.x - O.x)*(ll)(B.y - O.y) - (A.y - O.y)*(ll)(B.x - O.x);
+	}
+
+	vector<Point> convex_hull(vector<Point> P){
```

Comment:
Add as a comment the algorithm complexity
Example:
/*
  Complexity: O(n)  
*/